src/common/dom-helpers/dom-base.ts

```diff
@@ -0,0 +1,27 @@
+import {Renderer2, ElementRef} from '@angular/core';
```

Comment:
try to avoid `-helper`, `-base` and other meaningless prefixes

Reply:
according to this remark, files have been renamed in the commit: [blur directive, dom base class: refactoring](https://github.com/qgrid/ng2/pull/39/commits/bb9b43a5374e93ada17ace5c6f028fd5aa7ac651).